geth/chain.py

```diff
@@ -131,7 +131,11 @@ def write_genesis_file(
         }
 
     # Assign a signer (coinbase) to the genesis block for Clique
-    extraData = bytes("0x" + "0" * 64 + coinbase[2:] + "0" * 130, "ascii")
+    extraData = (
+        bytes("0x" + "0" * 64 + coinbase[2:] + "0" * 130, "ascii")
```

Comment:
There is another bug here that made me realize this problem to begin with but now just realizing there is another bug... because in theory, this change should not have affected the `ape-geth` plugin.

But I realize now that even  though coinbase may be set to the zero address here, the real sealer address that matters is used in geth kwargs `miner_etherbase`, thus that is the one that should be used in the calculation here.

It can be avoided with this PR though- because we can continue passing our own config in.

Reply:
I'm not 100% sure I understand... It sounds like the actual fix is to add `miner_etherbase` to the geth kwargs and set the default there?

Reply:
I am not really sure about the other bug... somehow needs to honor miner_etherbase for the calculation.

However... extraData as a kwarg to this function does nothing right now. you can pass in anything and it is always overridden. Pycharm also warns me of this. So they are separate bugs, one just revealed the other one.

Reply:
Ah, gotcha. Yeah, I don't have any good ideas on testing this really, but it's straightforward so will get it merged. Thanks for the PR!